src/stories/tests/HooksTests.stories.tsx

```diff
@@ -219,6 +219,7 @@ TypeSearchTermRenderSectionsCustomOrder.args = {
     },
   ],
 };
+// TODO: duplicate tests? these don't seem to run on npm run test-storybook
 TypeSearchTermRenderSectionsCustomOrder.play = async ({ canvasElement }) => {
```

Comment:
These do run but they're aren't using the Components we export. These classes are found in the `/src/stories/Autocomplete/Hook/index.tsx#HooksTemplate`. We should probably update them there to be consistent in terms of our docs.